manifests/init.pp

```diff
@@ -13,8 +13,20 @@
 #
 # $group::                         Override the name of the puppet group.
 #
+# $dir_prefix::                    Override the where to find the data directory on Windows.
```

Comment:
Good point. My thought for making it a param was that people can install things in alternate locations such as the D: drive, the problem is that I didn't code it that way. I am removing the param.